Docs changes for recent auth sidecar work (#339)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_accessing-uis-for-stf-components.adoc

[id="accessing-uis-for-stf-components_{context}"]

> = Accessing User Interfaces for {ProjectShort} Components

Reviewer: ```suggestion
= Accessing user interfaces for {ProjectShort} components
```

[role="_abstract"]

> In {OpenShift}, applications are exposed to the external network through a route. For more information about Routes, see https://docs.openshift.com/container-platform/{NextSupportedOpenShiftVersion}/networking/configuring_ingress_cluster_traffic/overview-traffic.html[Configuring ingress cluster traffic].

Reviewer: ```suggestion
In {OpenShift}, applications are exposed to the external network through a route. For more information about routes, see https://docs.openshift.com/container-platform/{NextSupportedOpenShiftVersion}/networking/configuring_ingress_cluster_traffic/overview-traffic.html[Configuring ingress cluster traffic].
```

> In {Project} ({ProjectShort}), HTTPS routes are exposed for each service that has a web based interface. These routes are protected by {OpenShift} RBAC and any user with a ClusterRoleBinding allowing them to view {OpenShift} Namespaces can log in. For more information about RBAC, see https://docs.openshift.com/container-platform/{NextSupportedOpenShiftVersion}/authentication/using-rbac.html[Using RBAC to define and apply permissions].

Reviewer: ```suggestion
In {Project} ({ProjectShort}), HTTPS routes are exposed for each service that has a web-based interface. These routes are protected by {OpenShift} RBAC and any user that has a `ClusterRoleBinding` that enables them to view {OpenShift} Namespaces can log in. For more information about RBAC, see https://docs.openshift.com/container-platform/{NextSupportedOpenShiftVersion}/authentication/using-rbac.html[Using RBAC to define and apply permissions].
```

Reviewer: Hi @csibbitt , a couple of Qs for you:
- Do you think there is a need to spell out RBAC? I don't think so, our users are super technical but I'd like your opinion please.
- Can we presume that our users will know what a ClusterRoleBinding is? I added back ticks to that but in hindsight I think that is wrong.

Author: Considering the audience and the fact that we immediate provide a link to additional documentation I don't think it's necessary.

Author: Backticks seem okay to me? I'm not 100% certain on the convention, but it's an object type name. The link to the official openshift docs are meant to clarify what a ClusterRoleBinding is and how they are managed. Since these are existing Openshift concepts that have nothing to do with STF my inclination is not to give much explanation here.
